docs/admin/maintenance/update_workstations.rst

```diff
@@ -44,6 +44,6 @@ out the new release: ::
   (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__).
 
 Finally, run the following commands: ::
-
+  sudo apt update
```

Comment:
Code formatting looks broken here?

Reply:
Ah yup, thanks - strange the linter didn't complain about that! Should be fixed now.